### ISSUE TITLE
swagger-codegen crashes when generating swift

### ISSUE THREAD

**Issue by bherila** [swagger-api/swagger-codegen#2538]
```
$ swagger-codegen-local generate -l swift -i midtier-swagger.json 
[main] INFO io.swagger.parser.Swagger20Parser - reading from midtier-swagger.json
Exception in thread "main" java.lang.RuntimeException: Could not process model 'CampaignApiModel'
	at io.swagger.codegen.DefaultGenerator.generate(DefaultGenerator.java:244)
	at io.swagger.codegen.cmd.Generate.run(Generate.java:223)
	at io.swagger.codegen.SwaggerCodegen.main(SwaggerCodegen.java:36)
Caused by: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
	at io.swagger.codegen.languages.SwiftCodegen.fromProperty(SwiftCodegen.java:322)
	at io.swagger.codegen.DefaultCodegen.addVars(DefaultCodegen.java:2232)
	at io.swagger.codegen.DefaultCodegen.addVars(DefaultCodegen.java:2206)
	at io.swagger.codegen.DefaultCodegen.addVars(DefaultCodegen.java:2193)
	at io.swagger.codegen.DefaultCodegen.fromModel(DefaultCodegen.java:1013)
	at io.swagger.codegen.DefaultGenerator.processModels(DefaultGenerator.java:799)
	at io.swagger.codegen.DefaultGenerator.generate(DefaultGenerator.java:237)
	... 2 more
```

this is building from latest source (same branch as my other PR)

input .json : https://gist.github.com/bherila/e664670b38d4d9c979b094bbcf0134ff (caution: it's big)

Unfortunately I am very novice at swift so I probably cannot be helpful in terms of PR fixing this one. But some community help would really be appreciated 😄

**Comment by wing328** [comment]
@bherila thanks for reporting the issue with the spec. We'll look into that ...

**Comment by wing328** [comment]
@bherila it's due to number used in enum values. I filed https://github.com/swagger-api/swagger-codegen/pull/2508 to support number in enum and will merge it on Monday.

Will let you know once it's merged.

**Comment by alecgorge** [comment]
@wing328 I am still seeing this exact same error on 2.2.0 even though #2508 seems to indicate that it should be fixed

**Comment by wing328** [comment]
@bherila @alecgorge fixed via #3535. Please pull the latest master to give it a try.

**Comment by alecgorge** [comment]
@wing328 yup, it works great. Thanks so much for the quick turn around.
